scripts/suite.py

```diff
@@ -172,11 +172,9 @@
                               in the output of teuthology-suite command. -1
                               for a random seed [default: -1].
  --force-priority             Skip the priority check.
- --disable-num-jobs-check     Skip the number of jobs check. By default,
-                              teuthology will not allow you to schedule more
-                              than JOBS_TO_SCHEDULE_THRESHOLD=500 jobs, because
-                              it is too high. Use this if you need to schedule
-                              more than 500 jobs.
+ --job-threshold <value>      Do not allow to schedule the suite if the number
+                              of jobs exceeds the given <value>. Use 0 to skip
+                              the check [default: {default_job_threshold}].
```

Comment:
Nit: The number of jobs in a suite is constant (in the hundreds of thousands for rados, I think).  What is being scheduled and where the number of jobs is variable is a run:
```suggestion
 --job-threshold <threshold>  Do not allow to schedule the run if the number
                              of jobs exceeds <threshold>. Use 0 for unlimited
                              [default: {default_job_threshold}].
```

Reply:
That makes sense as for run, but we should avoid the 'unlimited' or 'limit' key words because there is --limit options which limits the number of jobs to schedule, i.e. it anyways schedule job but with less jobs.